combat-trainer.lic

```diff
@@ -363,7 +366,15 @@ class LootProcess
         echo "Severity to Wounds: #{game_state.wounds}" if $debug_mode_ct
         do_necro_ritual(mob_noun, 'consume', game_state) unless game_state.wounds.empty?
       end
-      do_necro_ritual(mob_noun, @ritual_type, game_state) unless game_state.prepare_consume || (DRSkill.getxp('Thanatology') > 30 && @skin)
+      if !game_state.prepare_consume || !@skin
+        if DRSkill.getxp('Thanatology') <= 30
+          do_necro_ritual(mob_noun, @ritual_type, game_state)
+        elsif DRSkill.getxp('First Aid') <= 30 && @necro_train_first_aid
+          do_necro_ritual(mob_noun, @ritual_type, game_state)
+        end
```

Comment:
Since the content of each part of the if statement is the same we should probably condense them. for readability I'd suggest moving the checks out into a single helper method, something like

```
def should_perform_ritual
  DRSkill.getxp('Thanatology') <= 30 || (@necro_train_first_aid && DRSkill.getxp('First Aid') <= 30 )
end
```

Leaving us with 
```
do_necro_ritual(mob_noun, @ritual_type, game_state) if should_perform_ritual
```